frontend/src/components/PlayerPage.tsx

```diff
@@ -0,0 +1,117 @@
+import {Avatar, Card, Typography} from "@mui/material";
+import styled from "@emotion/styled";
+import usePlayer from "../hooks/usePlayer.ts";
+import {StyledButtonContainer} from "./styled/StyledButtonContainer.ts";
+import {StyledFormButton} from "./styled/StyledFormButton.ts";
+import {useNavigate, useParams} from "react-router-dom";
+import {useEffect, useState} from "react";
+import {useStore} from "../hooks/useStore.ts";
+import AvatarImage from "../assets/images/avatar.png";
+
+export default function PlayerPage() {
+    const [name, setName] = useState<string>("")
+    const params = useParams()
+    const navigate = useNavigate()
+    const player = usePlayer(name)
+    const user = useStore(state => state.user)
+    const logout = useStore(state => state.logout)
+
+    const isEnemy = user !== player?.name
+
+    useEffect(() => {
+        if (params.name) {
+            setName(params.name)
+            console.log("inside playerPage")
+        }
+    }, [params.name]);
+
+    if (typeof player !== "undefined") {
+        return (
+            <>
+            <StyledCard>
+                <StyledAvatar isEnemy={`${isEnemy}`} src={`${AvatarImage}`} sx={{width: 125, height: 125}}/>
```

Comment:
Der heutige Alibi-Kommentar: Warum `${}`? 🤔

Reply:
React sagt mir manchmal, dass boolean props für styled elements als strings übergehen werden müssen. Ich check nicht warum das so ist, aber so kommen keine Warnungen in der console.